Makefile

```diff
@@ -145,11 +145,11 @@ endif
 endif
 endif
 
-module: $(BIN_OUTPUT_PATH)/viamrtsp
-	tar czf $(BIN_OUTPUT_PATH)/module.tar.gz $(BIN_OUTPUT_PATH)/viamrtsp
+module.tar.gz: $(BIN_OUTPUT_PATH)/viamrtsp
```

Comment:
please change the target back to `module` and change it to use this pattern: https://github.com/erh/viamrtsp/commit/4d7999f204a58b6564cfe2c0386e330fbf6c6f5c